guides/common/modules/proc_creating-custom-ssl-certificates-for-smart-proxy-server.adoc

```diff
@@ -31,33 +31,43 @@ If you already have a private key for this {SmartProxyServer}, skip this step.
 default_bits       = 4096
 distinguished_name = req_distinguished_name
 req_extensions     = req_ext
+x509_extensions = usr_cert
```

Comment:
I don't think we need those (in both places!), as `x509_extensions` apply to the certificate, and that's what the CA produces based on our input, not what we produce using the commands in these procedures.